samples/nrf_desktop/sample.yaml

```diff
@@ -7,9 +7,21 @@ tests:
     build_on_all: true
     platform_whitelist: nrf52840_pca20041 nrf52840_pca10056 nrf52840_pca10059
     tags: bluetooth ci_build
+  test_debug_mcuboot:
+    build_only: true
+    build_on_all: true
+    platform_whitelist: nrf52840_pca20041 nrf52840_pca10059
+    tags: bluetooth ci_build
+    extra_args: "CMAKE_BUILD_TYPE=ZDebugMCUBoot"
```

Comment:
Does it only build the variant, or also try to load and run on HW?

Reply:
Pretty sure it only builds.

Reply:
@Qbicz , it only builds. @wiminordic is picking only the artifacts he needs for running on test machines.